scrapeGithub/database.py

```diff
@@ -0,0 +1,56 @@
+import sqlite3
+import pdb
+
```

Comment:
Please add comments what this file is for and how to use it.

Reply:
Ok.